remix/Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: up start_db
+
+up:
```

Comment:
I recommend renaming the commans to `start` and `stop` as it makes more sense to start and stop and application 😅

Reply:
Ok, I have fixed it